pkg/statistics/handle/bootstrap.go

```diff
@@ -40,6 +40,8 @@ import (
 	"go.uber.org/zap"
 )
 
+const initStatsStep = int64(500)
```

Comment:
Please add some comments to this magic number. Thanks!

Reply:
Thanks, I have updated it.